.travis.yml

```diff
@@ -9,6 +9,8 @@ matrix:
       services:
         - docker
       env: OSNAME=linux_openresty
+    - os: linux
```

Comment:
run this test case in Github Action, the Travis-CI is slow.

what do you think? @nic-chen

Reply:
It's good. I will try to move to github actions.

Reply:
I think we can merge this PR first, and then submit the PR separately to fix it. What do you think?